.github/workflows/reusable-build.yml

```diff
@@ -171,7 +171,7 @@ jobs:
         shell: bash
         run: |
           mkdir -p certs
-          if [[ ${{ github.event_name }}  == 'pull_request' ]]; then
+          if [[ "${{ github.event_name }}"  == 'pull_request' ]]; then
```

Comment:
Possible stupid question incoming:
Is there a difference between single quotes and double quotes in a comparison?

Reply:
Looks like only a difference if we're doing string interpolation, which we aren't here.  LGTM!

Reply:
single quotes don't expand in bash. It shouldn't matter, but given the goof on missing quotes on the left hand side I probably should just make them match

Reply:
I agree, it's not a bug. Only a style concern.